docs/ga-requirements.md

```diff
@@ -0,0 +1,51 @@
+P1 (e.g. cannot GA without these):
+* End-to-end tests
+  * OTLP, Jaeger and Zipkin
+  * Spring Boot and Wildfly
+    * (Wildfly chosen due to common javaagent issues around jboss modules and jboss logging)
+  * Java 8, 11, and the latest Java version
+* Benchmarking
+  * Runtime overhead benchmark
+  * Startup overhead benchmark
+* Implement all required semantic attributes
+  * Non-required semantic attributes are nice to have, but not required for GA
+* Basic configuration points
+  * Add custom auto-instrumentation
+  * Ability to build "custom distro"
+* Documentation (tag "documentation" minus tag "contributor experience")
+  * All configuration options
+    * Standard OpenTelemetry SDK + Exporter configuration options
+    * Auto-instrumentation configuration options (e.g. disable/enable, peer.service mapping)
+  * For each instrumentation
+    * Document any instrumentation-specific configuration
+  * Document the basic configuration points (above)
+    * How to write your own auto-instrumentation
+      * (much of this can be shared with contributor documentation below)
+    * How to build your own "custom distro"
+  * How to troubleshoot (start documenting common issues somewhere)
+* Library (manual) instrumentations for libraries commonly used with Spring
+  * Spring WebMVC, Spring WebFlux, Spring RestTemplate, Spring Kafka, Spring AMQP,
+    Reactor, JDBC, java.util.concurrent
+* Library (manual) instrumentations for libraries commonly used with Android
+  * OkHttp, gRPC
+
+P2:
+* Contributor experience (tag "contributor experience" plus tag "cleanup" plus tag "sporadic test failure")
+  * New contributor documentation
+    * How to write new instrumentation (auto, library, tests)
```

Comment:
But P1 already has "How to write your own auto-instrumentation". And I think that without these documentation here nobody will be able to "Add custom auto-instrumentation". But putting all "New contributor documentation" in P1 seems like too aggressive to me :(